apps/dashboard/app/views/files/index.html.erb

```diff
@@ -1,28 +1,28 @@
 <%# z-index:999 cause dropdowns are 1000 default sticky-top is 1020 https://getbootstrap.com/docs/4.6/layout/overview/#z-index %>
+
 <div class="text-right sticky-top bg-white border-bottom p-2 m-3" style="z-index: 999">
   <!-- terminal button href is set via JavaScript because the URL is based on the current directory -->
   <% if Configuration.files_enable_shell_button %>
     <%= render partial: 'shell_dropdown' %>
   <% end %>
 
-  <button id="new-file-btn" type="button" class="btn btn-outline-dark btn-sm"><i class="fas fa-plus" aria-hidden="true"></i> New File</button>
-  <button id="new-dir-btn" type="button" class="btn btn-outline-dark btn-sm"><i class="fas fa-folder-plus" aria-hidden="true"></i> New Directory</button>
-  <button id="upload-btn" type="button" class="btn btn-primary btn-sm"><i class="fas fa-upload" aria-hidden="true"></i> Upload</button>
-  <button id="download-btn" type="button" class="btn btn-primary btn-sm"><i class="fas fa-download" aria-hidden="true"></i> Download</button>
-  <button id="copy-move-btn" type="button" class="btn btn-outline-dark btn-sm"><i class="fas fa-copy" aria-hidden="true"></i> Copy/Move</button>
-  <button id="delete-btn" type="button" class="btn btn-danger btn-sm"><i class="fas fa-trash" aria-hidden="true"></i> Delete</button>
+    <button id="new-file-btn" type="button" class="btn btn-outline-dark btn-sm"><i class="fas fa-plus" aria-hidden="true"></i> New File</button>
+    <button id="new-folder-btn" type="button" class="btn btn-outline-dark btn-sm"><i class="fas fa-folder-plus" aria-hidden="true"></i> New Folder</button>
```

Comment:
Linux does not have folders, Linux has directories.  Folders are a Windows thing.  Probably best to use names that are from Linux since this is managing things on the Linux based filesystems.

Reply:
> Linux does not have folders, Linux has directories. Folders are a Windows thing. Probably best to use names that are from Linux since this is managing things on the Linux based filesystems.

Probably a bit too nit-picky.

Reply:
Maybe, but why change it then?  It was already referencing "dir" and "Directory" per Linux conventions.

Reply:
i changed it.